.gitmodules

```diff
@@ -1,6 +1,9 @@
-[submodule "closure-templates"]
+[submodule "deps/closure-templates"]
 	path = deps/closure-templates
 	url = https://github.com/google/closure-templates
 [submodule "deps/closure-library"]
 	path = deps/closure-library
-	url = https://github.com/google/closure-library
+	url = https://github.com/CarlosM10011/closure-library
```

Comment:
Why did you change where this submodule points to?

Reply:
I forked it because I changed closure/goog/events/browserfeature.js to use `goog.module`, instead of `goog.provide` and `goog.scope`. The old file using `goog.scope` would cause the compiler to throw a runtime exception with an internal compiler error. I'm not entirely sure why it is gone, but converting the file caused the error to vanish.

I submitted a PR at (https://github.com/google/closure-library/pull/1080).